src/kvstore/RocksEngine.cpp

```diff
@@ -32,7 +32,7 @@ class RocksWriteBatch : public WriteBatch {
     rocksdb::DB* db_{nullptr};
 
 public:
-    explicit RocksWriteBatch(rocksdb::DB* db) : db_(db) {}
+    explicit RocksWriteBatch(rocksdb::DB* db) : batch_(4096), db_(db) {}
```

Comment:
why is 4096?

Reply:
Just  experience dependence,  maybe use a flag here is better.